whois-query/src/test/java/net/ripe/db/whois/query/integration/AbuseCTestIntegration.java

```diff
@@ -239,4 +240,66 @@ public void brief_query_shows_abusemailbox_twice() {
         final String briefResponse = TelnetWhoisClient.queryLocalhost(QueryServer.port, "-b 193.0.0.0");
         assertThat(briefResponse, not(containsString("notshown@abuse.net")));
     }
+    @Test
+    public void should_not_lookup_abuseC_if_rpsl_source_do_not_match() {
+
+        databaseHelper.updateObject(RpslObject.parse( "inetnum:       173.0.0.0 - 173.255.255.255\n" +
```

Comment:
Don't make reference to "RIPE" as the source, but something like 'NON-TEST' or 'INVALID'